tests/wpt/mozilla/tests/mozilla/document_elementsFromPoint.html

```diff
@@ -0,0 +1,7 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<link rel="match" href="document_elementsFromPoint_ref.html">
+<script>
+    var elements = document.elementsFromPoint(10, 10);
```

Comment:
Instead of writing the test like this, since this is testing a crash, let's do something in the lines of:

```html
<!doctype html>
<meta charset="utf-8">
<title>document.elementsFromPoint does not crash due to a missing display list</title>
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
<script>
test(function() {
    document.elementsFromPoint(10, 10);
}, "doesn't crash");
</script>
```